OBSERVABILITY.md

```diff
@@ -0,0 +1,119 @@
+## Observability with OpenTelemetry
+
+This Cloud Spanner client supports [OpenTelemetry Traces](https://opentelemetry.io/), which gives insight into the client internals and aids in debugging/troubleshooting production issues.
+
+By default, the functionality is disabled. You shall need to add OpenTelemetry dependencies, and must configure and
+enable OpenTelemetry with appropriate exporters at the startup of your application:
+
+**Table of contents:**
+
+* [Observability](#observability)
+  * [Tracing](#tracing)
+    * [OpenTelemetry Dependencies](#opentelemetry-dependencies)
+    * [OpenTelemetry Configuration](#opentelemetry-configuration)
+    * [SQL Statement span annotation](#sql-statement-span-annotation)
+    * [OpenTelemetry gRCP instrumentation](#opentelemetry-grpc-instrumentation)
+    * [Tracing Sample](#tracing-sample)
+
+### Tracing
+
+#### OpenTelemetry Dependencies
+
+Use a trace exporter of your choice, such as Google Cloud Trace like below:
+
+```javascript
+const {
+  TraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+const exporter = new TraceExporter();
+```
+
+#### OpenTelemetry Configuration
+
+```javascript
+const {NodeSDK} = require('@opentelemetry/sdk-node');
+const {Resource} = require('@opentelemetry/resources');
+const {
+  NodeTracerProvider,
+  TraceIdRatioBasedSampler,
+} = require('@opentelemetry/sdk-trace-node');
+const {
+  BatchSpanProcessor,
+} = require('@opentelemetry/sdk-trace-base');
+const {
```

Comment:
Remove the resource and these attributes as these were deprecated and not working